apps/frontend-manage/src/components/questions/tags/TagActions.tsx

```diff
@@ -0,0 +1,58 @@
+import { faPencil, faTrash } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Tag } from '@klicker-uzh/graphql/dist/ops'
+import { Button, ThemeContext } from '@uzh-bf/design-system'
+import { useContext } from 'react'
+import { twMerge } from 'tailwind-merge'
+import TagDeletionModal from './TagDeletionModal'
+
+interface TagActionsProps {
+  tag: Tag
+  active: boolean
+  setEditMode: (editMode: boolean) => void
+  isDeletionModalOpen: boolean
+  setIsDeletionModalOpen: (isDeletionModalOpen: boolean) => void
+}
+
+function TagActions({
+  tag,
+  active,
+  setEditMode,
+  isDeletionModalOpen,
+  setIsDeletionModalOpen,
+}: TagActionsProps) {
+  const theme = useContext(ThemeContext)
+
+  return (
+    <div className="flex-row hidden text-black group-hover:flex">
+      <Button
+        basic
+        disabled={active}
```

Comment:
Editing and deleting Tags is only possible when they are not active. Otherwise, strange race conditions are created and questions are filtered for tags that do not exist anymore in this form (and can therefore not even be de-selected).